README.md

```diff
@@ -35,176 +45,6 @@ The table below summarizes the results of running various ML models through our
 
 ***
 
-### Op conversion status per model
-
-#### Mnist (Eval)
-| aten ops                             | status   |   count |
-|:-------------------------------------|:---------|--------:|
-| aten._log_softmax.default            | ✘        |       1 |
-| aten.addmm.default                   | ✅       |       2 |
-| aten.clone.default                   | ✅       |       2 |
-| aten.convolution.default             | ✘        |       2 |
-| aten.max_pool2d_with_indices.default | ✘        |       1 |
-| aten.relu.default                    | ✅       |       3 |
-| aten.t.default                       | ✅       |       2 |
-| aten.view.default                    | ✅       |       1 |
```

Comment:
If we remove these, we lose valuable info from reports, right?
Lets not remove till we gain that in cumulative and model pages

Reply:
Added these tables back. Only models that compile and run will show up.